sdk/keyvault/azure-security-keyvault-keys/inc/azure/keyvault/keys/delete_key_operation.hpp

```diff
@@ -108,6 +109,30 @@ namespace Azure { namespace Security { namespace KeyVault { namespace Keys {
      * @return std::string
      */
     std::string GetResumeToken() const override { return m_continuationToken; }
+
+    /**
+     * @brief Create a #DeleteKeyOperation from a previously created \p resumeToken updated to the
+     * the latest operation status.
+     *
+     * @remark After the operation is init, it is used to poll the last update from the server using
+     * the \p context.
+     *
+     * @param client A #KeyClient that is used for getting status updates.
+     * @param resumeToken A previously generated token.
+     * @param context A #Azure::Core::Context controlling the request lifetime.
+     * @return DeleteKeyOperation
+     */
+    static DeleteKeyOperation CreateFromResumeToken(
+        Azure::Security::KeyVault::Keys::KeyClient const& client,
+        std::string const& resumeToken,
```

Comment:
Swap the order of parameters here, and in docs, and elsewhere. This will also match what .NET does.